Вторая часть по массивам.

Boroda9992 opened the pull request.

Boroda9992 committed:
Вторая часть по массивам.

dmitry-bizin commented on src/Main.java:
```diff
@@ -2,18 +2,78 @@
+        System.out.println("__________________________________________________________________________________________");
+
+        int sum = 0;
+        for (int i : arr) sum = sum + i;
```
фигурные скобки надо ВСЕГДА писать после if/else/for/while, лучше сразу приучаться писать правильно, иначе потом всё равно придётся переучиваться. Также переменную лучше назвать не одной буквой, потому что обычно так называют индексы массива, а здесь уже не индекс, а сам элемент

Boroda9992 replied:
поправил

dmitry-bizin commented on src/Main.java:
```diff
@@ -2,18 +2,78 @@
+        System.out.println("__________________________________________________________________________________________");
+
+        int sum = 0;
+        for (int i : arr) sum = sum + i;
```
фигурные скобки надо ВСЕГДА писать после if/else/for/while, лучше сразу приучаться писать правильно, иначе потом всё равно придётся переучиваться. Также переменную лучше назвать не одной буквой, потому что обычно так называют индексы массива, а здесь уже не индекс, а сам элемент

Boroda9992 replied:
поправил

dmitry-bizin commented on src/Main.java:
```diff
@@ -2,18 +2,78 @@
+        int sum = 0;
+        for (int i : arr) sum = sum + i;
+
+        int average = sum / arr.length;
```
деление с потерей остатка

Boroda9992 replied:
поменял int на double

Boroda9992 committed:
Вторая часть по массивам.
28, 62 - добавил фигурные скобки после for
66 - поменял тип данных на double, чтобы не терять остаток при вычислении среднего
